### PR TITLE
python for linear algebra

### PR DESCRIPTION
**Pull request by santoshreddy254**

### CODE REVIEW

File: content/math/python_for_linera_algebra_Q.ipynb [hunk 1]
```diff
@@ -0,0 +1,663 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Go through the previous content of python for linear algebra\n",
+    "In this we focus on \n",
+    "* ndarray\n",
+    "* Arthematic operations\n",
```
Comment (alex-mitrevski) [inline content/math/python_for_linera_algebra_Q.ipynb]: Arithmetic

File: content/math/python_for_linera_algebra_Q.ipynb [hunk 2]
```diff
@@ -0,0 +1,663 @@
...
+    "n = 3\n",
+    "m = 2\n",
+    "arr1[:n,:m]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Arthematic operations"
```
Comment (alex-mitrevski) [inline content/math/python_for_linera_algebra_Q.ipynb]: Arithmetic

File: content/math/python_for_linera_algebra_Q.ipynb [hunk 3]
```diff
@@ -0,0 +1,663 @@
...
+   "metadata": {},
+   "source": [
+    "#### Go through the previous content of python for linear algebra\n",
+    "In this we focus on \n",
+    "* ndarray\n",
+    "* Arthematic operations\n",
+    "* Indexing\n",
+    "* Slicing\n",
+    "\n",
+    "## Different ways to init and declare a numpy array"
```
Comment (alex-mitrevski) [inline content/math/python_for_linera_algebra_Q.ipynb]: Please spell out "initialise" completely

File: content/math/python_for_linera_algebra_Q.ipynb [hunk 4]
```diff
@@ -0,0 +1,663 @@
...
+    "arr1[:n,:]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Extract cloumns\n",
```
Comment (alex-mitrevski) [inline content/math/python_for_linera_algebra_Q.ipynb]: columns

File: content/math/python_for_linera_algebra_Q.ipynb [hunk 5]
```diff
@@ -0,0 +1,663 @@
...
+   "metadata": {},
+   "source": [
+    "## Arthematic operations"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Application of what we learnt till now"
```
Comment (alex-mitrevski) [inline content/math/python_for_linera_algebra_Q.ipynb]: learnt till -> learned until

File: content/math/python_for_linera_algebra_Q.ipynb (hunk 1 above)
Comment (santoshreddy254) [reply content/math/python_for_linera_algebra_Q.ipynb]: chnaged

File: content/math/python_for_linear_algebra_exercises.ipynb [hunk 6]
```diff
@@ -308,7 +322,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Application of what we learnt till now"
+    "# Application of what we learnt untill now"
```
Comment (alex-mitrevski) [inline content/math/python_for_linear_algebra_exercises.ipynb]: Misspelled until

Comment (santoshreddy254) [reply content/math/python_for_linear_algebra_exercises.ipynb]: done

File: content/math/python_for_linera_algebra_Q.ipynb (hunk 4 above)
Comment (santoshreddy254) [reply content/math/python_for_linera_algebra_Q.ipynb]: columns

File: content/math/python_for_linera_algebra_Q.ipynb (hunk 2 above)
Comment (santoshreddy254) [reply content/math/python_for_linera_algebra_Q.ipynb]: Arithmetic

**Pull request merged**